modules/apps/foundation/frontend-theme/responsive-boat-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,278 @@
+/* These inject tags are used for dynamically creating imports for themelet styles, you can place them where ever you like in this file. */
+
+/* inject:imports */
+/* endinject */
+
+/* This file allows you to override default styles in one central location for easier upgrade and maintenance. */
+
+
+/********** VARIABLE SECTION **********/
+$expandTime: 0.4s; 				// time it takes for mobile navbar to expand or collapse
+$fadeTime: 0.5s; 				// time it takes for navbar links to change color when hovered over
+$navBtnPad: 20px; 				// padding above and below nav-button
+$navBtnHeight: 40px; 			// height of nav-button
```

Comment:
Do we really need comments for these variables?  The variable name should be detailed enough to provide this information.  If it is not, we should rename the SASS variable.